portfolio/src/main/webapp/script.js

```diff
@@ -216,3 +195,103 @@ function initializeAuth() {
       }
     });
 }
+
+/**
+ * Creates the control component that shows information about tropical storms
+ * @param {HTMLDivElement} controlDiv
+ */
+function InfoControl(controlDiv) {
+  let controlUI = document.createElement("div");
+  controlUI.classList.add("tropicalStormInfoControl");
+  controlDiv.appendChild(controlUI);
+
+  let controlText = document.createElement("div");
+  controlText.textContent =
+    "Hover over a path to get info on the tropical storm.";
+  controlUI.appendChild(controlText);
+}
+
+/**
+ * Generates a map from Google Map API
+ */
+function initializeMap() {
+  fetch("/tropical-storm-data")
+    .then((response) => response.json())
+    .then((tropicalStormData) => {
+      map = new google.maps.Map(document.getElementById("map"), {
+        center: { lat: 35.5733225, lng: -80.7187387 },
```

Comment:
I think you could potentially use HTML5's Geolocation API to get user's current location[1] and set as the center point on map, instead of hardcoding this initial center point. 

[1] https://developers.google.com/maps/documentation/javascript/geolocation

Reply:
In this case hardcoding the location coordinates is my desired result